example/image-classification/train_imagenet.py

```diff
@@ -56,6 +54,8 @@ def set_imagenet_aug(aug):
         dtype            = 'float32'
     )
     args = parser.parse_args()
```

Comment:
Maybe we should rearrange line 56-58? It looks like set_imagenet_aug() does nothing on args.